content/markdown/concepts/emphasis/emphasis.md

```diff
@@ -41,16 +41,16 @@ To italicize text, add one asterisk `*` or underscore `_` before and after a wor
 _italic text_
 ```
 
-```markdown
-_italic text_
-```
-
 For example:
 
 ```markdown
-This avacado egg sandwich is _very_ good.
+This avocado egg sandwich is _very_ good.
 ```
 
+The output would be:
+
+This avocado egg sandwich is _very_ good.
+
 ## Strikethrough
 
 To make a strikethrough text, use 2 tildes `~` before and after a piece of text.
```

Comment:
```suggestion
A strikethrough can be applied to text by surrounding with two tildes `~~`, like in the example below:
```

And then I would consolidate the following two snippets into one that looks like this:

```md
I did ~~not~~ build a website.
```